.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*    @ganeshrockz @ds-ms
\ No newline at end of file
+*    @ganeshrockz @ds-ms @rsamigullin
```

Comment:
Just remove them because they no longer own this action. You can add me instead. @OliverMKing